src/gui/dialogs/game_stats.cpp

```diff
@@ -68,7 +68,7 @@ unit_const_ptr game_stats::get_leader(const int side)
 static std::string controller_name(const team& t)
 {
 	static const std::array<t_string, 3> names {{_("controller^Human"), _("controller^AI"), _("controller^Idle")}};
-	return "<span color='#808080'><small>" + names[t.controller().v] + "</small></span>";
+	return "<span color='#808080'><small>" + names[t.controller()] + "</small></span>";
```

Comment:
The game crashes when opening the stats dialog. Note the line preceding it, vs the changed indexes in the new enum.

Reply:
Fixed.